### DISCUSSION TITLE
Variety in lemmatizer output

### DISCUSSION THREAD

**Discussion by spatiebalk** [explosion/spaCy#10632] [Help: Coding & Implementations] [answered]
I am training a NER model that also needs to have a lemmatizer. This lemmatizer I initialized with the source model en_core_web_sm as that is fine and does not need to be trained in my use case.  The NER part works well, however, I have some questions about some of the behaviour of the lemmatizer.

1. I initialized the lemmatizer with the one from en_core_web_sm, and froze it during training, however I see that sometimes my trained model assigns a different lemma to a token then the en_core_web_sm model does.
2. For my trained model (and not trained lemmatizer) I see a varying performance of the lemmatizer where the same token gets different lemma values in different sentences. 

So I was wondering, can this behaviour be explained by the lemmatizer being context dependent or are these artifacts of some mistake during training?
For completeness, I have added the [config.cfg](https://github.com/explosion/spaCy/files/8450668/config.txt) file I have used.

**Comment by thomashacker** [comment, marked as answer]
Hello,
you also need to freeze the `tok2vec` component, since all other components sourced from `en_core_web_sm` are listening to it.
The lemmatizer depends on the `tagger`, so you can also verify whether the POS tags are also correct.
